Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine3.14
+FROM node:14-alpine3.14
```

Comment:
Node version 16 has been giving errors when trying to use docker compose. This version works best.